### PR TITLE
created endpoint for selected employee in server and fixed login and …

### PR DESCRIPTION
**Pull request by TsvAliaksandra**
…signup ui

### CODE REVIEW

File: client/app/components/pages/employee-career-day-page/CareerDayForManagerPage/CareerDayForManagerPage.tsx [hunk 1]
```diff
@@ -140,12 +144,12 @@ class CareerDayForManagerPage extends React.Component<
...
         objective={item}
         userRole={this.props.user.Roles}
-        handleSaveObjective={(objective: IUpdateObjectiveManager) =>
-          this.handleSaveObjective(objective)
-        }
-        handleDeleteObjective={(e: React.MouseEvent<HTMLElement>, objectiveId: number) =>
-          this.handleClickOnDeleteButton(e, objectiveId)
-        }
+        archived={this.props.selectedCareerDay.Archived}
+          handleSaveObjective={(objective: IUpdateObjectiveManager) => this.handleSaveObjective(objective)}
```
Comment (a-stasevich) [inline client/app/components/pages/employee-career-day-page/CareerDayForManagerPage/CareerDayForManagerPage.tsx]: Excess space

File: server/Domain/Services/Impl/UserService.ts [hunk 2]
```diff
@@ -67,4 +67,16 @@ export class UserServise implements IUserService {
     const encrtyptedEmail = encodeURIComponent(this._cryptoService.encrtyptAES(email));
     return `${this._hostname}/api/users/verifyEmail/${encrtyptedEmail}`;
   }
+
+  public async loginAsEmployee() {
```
Comment (Rezdoom) [inline server/Domain/Services/Impl/UserService.ts]: delete this func

File: server/Domain/Services/Impl/UserService.ts [hunk 3]
```diff
@@ -67,4 +67,16 @@ export class UserServise implements IUserService {
...
+    const user = await this._userRepository.findById(1);
+    if (user) {
+      return user;
+    }
+    return false;
+  }
+
+  public async selectedEmployee(id: number) {
```
Comment (Rezdoom) [inline server/Domain/Services/Impl/UserService.ts]: handle data and send user to client without secure data (password, email)

File: client/app/components/pages/employee-career-day-page/objective/index.tsx [hunk 4]
```diff
@@ -226,16 +218,22 @@ class Objective extends React.Component<IProps & WithStyles<ComponentClassNames>
             </Typography>
           </div>
 
-          <div style={{ padding: 0 }}>{this.renderObjectiveOptions()}</div>
+          <div style={{ padding: 0 }}>
```
Comment (a-stasevich) [inline client/app/components/pages/employee-career-day-page/objective/index.tsx]: Is should be moved to classes

File: client/app/components/pages/employee-career-day-page/objective/index.tsx [hunk 5]
```diff
@@ -118,37 +117,30 @@ class Objective extends React.Component<IProps & WithStyles<ComponentClassNames>
       <ExpansionPanel>
         <ExpansionPanelSummary>
           <div className={this.props.classes.alignFrom}>
-            {this.props.userRole === 'manager' ? (
+            {this.props.userRole === 'manager' ?
               [
```
Comment (CeQadrat) [inline client/app/components/pages/employee-career-day-page/objective/index.tsx]: You can use React.Fragment here

**Pull request merged**